src/components/Feedback/Bad.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+export default function Bad({size=16,fill='black',...props}) {
```

Comment:
The `Bad`, `Good`, `Normal` ... files have the different space with other files. They should be the same as the others